### ISSUE TITLE
[dexter-monitor] Column "Number of developers finished with installation" is empty

### ISSUE THREAD

**Issue by KarolAntczak** [Samsung/Dexter#88]
Column "Number of developers finished with installation" is empty, although I have multiple users listed in "User List" page. 

Moreover, there is an error thrown on browser console: 

```
angular.js:13920TypeError: Cannot read property 'toLocaleString' of null
    at UserService.getUserStatus.then (overviewCtrl.js:145)
    at angular.js:16383
    at m.$eval (angular.js:17682)
    at m.$digest (angular.js:17495)
    at m.$apply (angular.js:17790)
    at l (angular.js:11831)
    at J (angular.js:12033)
    at XMLHttpRequest.t.onload (angular.js:11966)
```

**Comment by marchpig** [comment]
Did you set `DexterUserList` table?

**Comment by marchpig** [comment]
Fixed this issue - 9cf98f9ed8354869f76e90828b59930bcfecae7e
